### DISCUSSION TITLE
[Feature request]: add Guilde-64 mid and interpreter

### DISCUSSION THREAD

**Discussion by ghost** [mupen64plus-ae/mupen64plus-ae#1025] [Ideas]
Describe the feature
So made 2 custom guide 64 configure which is call Guilde 64 mid and Guilde 64 interpreter. So guys added this official build because interpreter need for sin and punishment english patch rom hack to load up. And mid has read Framebuffer every frame turn on fixed mario kart 64 pc  Monitor glitch.
Both use Guilde 64 10th anniversary ver of pugins.

Add screenshots

![161436802-1ba0aa59-b27d-4040-8758-d09be9d3b5f3](https://user-images.githubusercontent.com/102778335/161437289-a69a8b7f-9c93-4dc9-bae0-b4c408614c84.jpg)
Here settings changed for both.
![Screenshot_20220403-121422_M64Plus FZ Pro](https://user-images.githubusercontent.com/102778335/161437403-44a9ad80-0427-4c1c-94cf-1c9b7684aa6a.jpg)
![Screenshot_20220403-121444_M64Plus FZ Pro](https://user-images.githubusercontent.com/102778335/161437404-d151fdef-90a8-465f-8f17-43cd1f52e378.jpg)
So games fixed
Guilde 64 mid
Mario kart 64
Fixed  Monitor in 1st track
Guilde 64 interpreter
Sin and punishment english patch rom
Allowed game to start up.

**Comment by fzurita** [comment]
Read backs are extremely slow when performed by the Glide64mk2 video plugin. You can already achieve the same result by using the GLideN64-Very-Accurate emulation profile.

**Reply by ghost** [reply]
No profile I used guide 64 10th anniversary and used read Framebuffer every frame and did not sutter in game unlike very accurate profile did so just ask for add mid profile into emulator.

**Comment by fzurita** [comment]
For sin and punishment, this sounds like a bug in the dynarec and it should be fixed instead of just making a new profile for it.

Can you provide a link to the translation patch?

**Reply by ghost** [reply]
Sin and punishment english patch was a whole rom and also english texture pack did not work. So can not link to it because rom not  just english patch but this file name.
Sin and Punishment - Successor of the Earth (U) [!].

**Comment by ghost** [comment]
Also patch rom was working on 2016 fz built.

**Comment by fzurita** [comment]
I was able to find find the patch.

http://micro-64.com/database/patches.shtml

**Comment by fzurita** [comment]
I created an issue in the core. It can be found here: https://github.com/mupen64plus/mupen64plus-core/issues/943 And we can hope that it gets fixed.

**Reply by ghost** [reply]
Try very Accurate profile for mario kart 64 but sutter on s10 plus the Guilde-64 mid profile made fixed graphics bug without sutter maybe name Guilde-64 10th anniversary accurate but profile should Guilde-64 mid should add because best one for mario kart 64.

**Reply by fzurita** [reply]
The stutter should go away after the first time you run it.

**Reply by ghost** [reply]
It not some use Guilde 64 10th anniversary so not profile and does not sutter so good profile add to emulator it self.
